src/modules/networks/Networks.ts

```diff
@@ -364,13 +237,13 @@ export class Networks {
       throw new UnsupportedNetwork(`Unsupported network id: ${chainId}`);
     }
     return {
-      chainId,
+      chainId: valueToHex(chainId),
```

Comment:
isn't `chainId` parameter already a string?

Reply:
Yes! I will do it more reliable